.prettierrc.js

```diff
@@ -12,4 +12,22 @@ module.exports = {
   jsxBracketSameLine: false, // default
   arrowParens: 'avoid', // default
   endOfLine: 'lf',
+  plugins: ['@ianvs/prettier-plugin-sort-imports'],
+  importOrder: [
+    '^(react)(.*)$',
+    // Special case for @ebay/nice-modal-react.
+    '^@ebay/nice-modal-react$',
+    '<THIRD_PARTY_MODULES>',
+    '^@(?!ebay/nice-modal-react)(.*)$',
+    '',
+    '^/app/(?!assets)(.*)$',
+    '',
+    '^[./](?!.*\\.(png|jpg|jpeg|gif|svg|webm|mp4)$)',
+    '',
+    '<TYPES>',
+    '',
+    '^/app/assets/.*\\.(png|jpg|jpeg|gif|svg|webm|mp4)$',
+    '.*\\.(png|jpg|jpeg|gif|svg|webm|mp4)$',
```

Comment:
```suggestion
    '.^[./](.*\\.(png|jpg|jpeg|gif|svg|webm|mp4)$)',
```
this condenses 29 and 30 I think - we don't really care if it's in assets or not. or if we do, we _only_ care that it's in assets, and not what kind of file it is, so maybe also add `'^/.*/assets/.*$'`